### DISCUSSION TITLE
Firefox extension still on 1.21 version

### DISCUSSION THREAD

**Discussion by pedrozimmermannsoares** [tprouvot/Salesforce-Inspector-reloaded#383] [General]
I use most firefox and I see that Salesforce Inspector Reloaded still are on 1.21 version

**Comment by tprouvot** [comment]
Hi @rotivordep 
You're right I need to release the 1.22 for Firefox !
I'll try to do this next week

**Comment by tprouvot** [comment]
Hi @rotivordep,
The version 1.22 is available for firefox, thanks for the highlight !

**Comment by kmacmillan25** [comment]
Hello, is it possible to update the Firefox extension to the new 1.23 release please? Thanks!

**Comment by tprouvot** [comment]
Hi @kmacmillan25 sure I'll do this before Thursday ! 
My bad about the delays, I have to automate firefox release and it will be done very soon !

**Reply by kmacmillan25** [reply]
Thanks very much @tprouvot ! I'm eager to use the new release!
